src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
+    if (isEmpty(data)) throw new HttpException(400, "Groups don't exist");
+    
```

Comment:
Missing existence condition here for the http code exception thrown.
Example: 
if isEmpty(data) OR !!this.users. findFirst({where: {id: userid}}) then throw new HttpException(400, "Groups don't exist");